onnxruntime/core/optimizer/graph_transformer.cc

```diff
@@ -7,8 +7,8 @@ using namespace ::onnxruntime::common;
 
 namespace onnxruntime {
 
-Status GraphTransformer::Apply(Graph& graph, bool& modified) const {
-  // The Graph should be in a good state prior this being called, so there should be no need to call Resolve here.
+Status GraphTransformer::Apply(Graph& graph, bool& modified, const std::vector<std::string>& ) const {
```

Comment:
You do not use the providers currently?

Reply:
No - I will add it in next PR. This will include changing all the transformers to check the provider to which the node is assigned prior. In this PR I am not enabling these transformers.... So it wont break anything in production